package.json

```diff
@@ -31,6 +31,7 @@
     "chai-datetime": "1.5.0",
     "chai-shallow-deep-equal": "1.4.6",
     "clean-webpack-plugin": "^3.0.0",
+    "compression-webpack-plugin": "^8.0.1",
```

Comment:
https://github.com/linagora/esn-frontend-inbox/blob/main/package.json#L74
are you sure of the version ?

Reply:
hmm I'm using the latest version, let me check if there is no conflict with our webpack version